Borysov/homework 5. Bootstrap, ajax/Quiz/src/main/java/borysov/controller/QuizController.java

```diff
@@ -0,0 +1,95 @@
+package borysov.controller;
+
+import borysov.entity.*;
+import borysov.exception.DAOException;
+import borysov.extractor.impl.ExtractorQuationFormImpl;
+import borysov.service.QuizService;
+import borysov.extractor.ExtractorQuationForm;
+import borysov.service.UserService;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.context.request.WebRequest;
+
+import javax.servlet.ServletRequest;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@Controller
+public class QuizController {
+    private static final Logger LOGGER = Logger.getLogger(QuizController.class);
+    @Autowired
+    QuizService service;
+
+    @GetMapping(value = "showQuizzes")
+    public String showQuizzes(Model model) {
+        LOGGER.info("ShowQuizzes");
+        model.addAttribute("listOfQuizzes", service.getListOfQuizzes());
+        return "showQuizzes";
+    }
+
+    @DeleteMapping(value = "delete/{id}")
+    public String getDelete(@PathVariable("id") int id) {
+        LOGGER.info("DeleteQuiz");
+
+        service.removeQuizById(id);
+        return "redirect:showQuizzes";
+    }
+
+    @RequestMapping(method = POST, value = "addQuiz")
+    public String addQuiz(WebRequest req, HttpSession session) {
+        LOGGER.info("AddQuiz");
+
+        String subject = req.getParameter("subject_field");
+        String theme = req.getParameter("theme_field");
+        User author = (User) session.getAttribute("currentUser");
+
+        Quiz quiz = new Quiz(subject, theme, author.getLogin());
+        service.addQuiz(quiz);
+
+        return "redirect:showQuizzes";
+    }
+
+    @RequestMapping(method = GET, value = "addQuestionAndAnswers")
+    public String getAddPage(WebRequest req, HttpSession session) {
+        LOGGER.info("getAddPage");
+        String quizId = req.getParameter("quiz_id");
```

Comment:
quiz_id can be provided through url. And setting attribute must not be to session